tests/Authentication.py

```diff
@@ -37,39 +37,60 @@ def testNoAuthentication(self):
         self.assertEqual(g.get_user("jacquev6").name, "Vincent Jacques")
 
     def testBasicAuthentication(self):
-        g = github.Github(self.login, self.password)
+        g = github.Github(self.login.login, self.login.password)
         self.assertEqual(g.get_user("jacquev6").name, "Vincent Jacques")
 
     def testOAuthAuthentication(self):
-        g = github.Github(self.oauth_token)
+        g = github.Github(self.oauth_token.token)
         self.assertEqual(g.get_user("jacquev6").name, "Vincent Jacques")
 
     def testJWTAuthentication(self):
-        g = github.Github(jwt=self.jwt)
-        self.assertEqual(g.get_user("jacquev6").name, "Vincent Jacques")
-
-    def testUserAgent(self):
-        g = github.Github(user_agent="PyGithubTester")
+        g = github.Github(jwt=self.jwt.token)
         self.assertEqual(g.get_user("jacquev6").name, "Vincent Jacques")
```

Comment:
Any assertions you want to put in here around verifying that the warning is logged?

Reply:
done